test/WebJobs.Script.Tests/Configuration/ScriptJobHostOptionsSetupTests.cs

```diff
@@ -230,50 +196,44 @@ public void Configure_AppliesRetry_ExponentialBackOffDelay(string expectedStrate
             Assert.Equal(maxIntervalTimeSpan, options.Retry.MaximumInterval);
         }
 
-        [Fact]
-        public void Configure_TimeoutDefaultsNull_IfNotDynamic()
-        {
-            var options = GetConfiguredOptions(new Dictionary<string, string>());
-            Assert.Equal(ScriptJobHostOptionsSetup.DefaultFunctionTimeout, options.FunctionTimeout);
-
-            // When functionTimeout is set as null
-            var settings = new Dictionary<string, string>
-            {
-                { ConfigurationPath.Combine(ConfigurationSectionNames.JobHost, "functionTimeout"), string.Empty }
-            };
-
-            options = GetConfiguredOptions(settings);
-            Assert.Equal(ScriptJobHostOptionsSetup.DefaultFunctionTimeout, options.FunctionTimeout);
-        }
-
-        [Fact]
-        public void Configure_NoMaxTimeoutLimits_IfNotDynamic()
+        [Theory]
+        [InlineData(ScriptConstants.ElasticPremiumSku)]
+        [InlineData(ScriptConstants.FlexConsumptionSku)]
```

Comment:
This is the main behavioral change - ensuring FlexConsumption timeouts can be unbounded